briefing.sqf

```diff
@@ -175,27 +161,22 @@ if (_unitfaction in ["civ_f", "civ_idap_f"]) exitwith {
 
 #include "f\briefing\f_briefing_civ.sqf"
 
-// DEBUG
-	if (f_param_debugMode == 1) then
-	{
-	player sideChat format ["DEBUG (briefing.sqf): Briefing for %1 slot selected.",_unitfaction];
-	};
+	// DEBUG
+	[_unitfaction] call _fnc_debug;
 };
 
 // ====================================================================================
 
 // BRIEFING: ZEUS
-// The following block of code executes only if the player is in a ZEUS (Gamelogic) slot; it automatically includes a file which contains the appropriate briefing data.
+// The following block of code executes only if the player is in a ZEUS (Gamelogic) 
+// slot; it automatically includes a file which contains the appropriate briefing data.
 
-if (_unitfaction == "") exitwith {
+if (_unitfaction == "" && ! (typeOf player isEqualTo "VirtualSpectator_F")) exitwith {

```

Comment:
If the player is a virtual spectator they now always see the "faction not defined" debug message.